src/components/Dashboard/PostsContainer.js

```diff
@@ -0,0 +1,52 @@
+/*global HM*/
+import React from 'react';
+import Post from './Post';
+import withFetch from '../../utils/withFetch';
+import { compose } from 'recompose';
+import Dismissable from '../Dismissable';
+import { WidgetContainer, WidgetHeader, WidgetBody, WidgetControls } from './Widget';
+
+const Posts = props => {
+	return (
+		<WidgetContainer name="posts">
+			<WidgetHeader title="posts">
+				<WidgetControls>
+					<form action="">
+						<select className="Select" name="" id="">
+							<option value="">Published</option>
+						</select>
+						<select className="Select" name="" id="">
+							<option value="">Latest</option>
+						</select>
+					</form>
+				</WidgetControls>
+			</WidgetHeader>
+			<WidgetBody>
+				<Dismissable name="notification" key="notification">
```

Comment:
Just wondering how to make this come back when there's new comments. I'd say make it's display conditional on `props.comments.data.length` and set the name to the current timestamp ``comments-notification-${Date.now()}``. The `name` prop is too generic right now.